recipes-app/iot2050-firmware-update/files/iot2050-firmware-update

```diff
@@ -571,16 +655,24 @@ def main(argv):
         if not erase_env_input == "y":
             sys.exit(1)
     try:
-        if args.force:
+        if args.force or args.firmware == None:
             # When forced, ignore all the checking and call the underlying updater directly
             updater = FirmwareUpdate(args.firmware)
         else:
```

Comment:
This introduce the constrian of the -b and firmware is None.

Better to have a subclass of the FU, maybe named with RollbackUpdate, then put the change into this sub-class, then here you can just call the sub-class by checking if args.rollback.